aws/main.tf

```diff
@@ -50,12 +53,24 @@ module "autoscaler" {
   cluster_name = var.cluster_name
 }
 
+
+# EFS and CSI driver should be deployed together
+module "efs" {
+  #count        = var.modules_info.efs.create ? 1 : 0
```

Comment:
Return it or remove all var.modules_info

Reply:
the question is if I'm going to need it for pre-requisite-dev and prod. I can't just remove it, no?

Reply:
If you will replace the prerequisite stack with it. you will need to import for the EFS